doc/contributing/maintaining/maintaining-dependencies.md

```diff
@@ -8,29 +8,30 @@ directories to create the Node.js binaries.
 All dependencies are located within the `deps` directory.
 This a list of all the dependencies:
 
-* [acorn][]
-* [ada][]
-* [base64][]
-* [brotli][]
-* [c-ares][]
-* [cjs-module-lexer][]
+* [acorn 8.8.2][]
+* [ada 2.5.0][]
+* [base64 0.5.0][]
+* [brotli 1.0.9][]
+* [c-ares 1.19.0][]
+* [cjs-module-lexer 1.2.2][]
 * [corepack][]
 * [googletest][]
-* [histogram][]
-* [icu-small][]
-* [llhttp][]
-* [minimatch][]
-* [nghttp2][]
-* [ngtcp2][]
-* [npm][]
-* [openssl][]
-* [postject][]
-* [simdutf][]
-* [undici][]
-* [uv][]
-* [uvwasi][]
-* [V8][]
-* [zlib][]
+* [histogram 0.11.8][]
+* [icu-small 73.1][]
+* [libuv 1.44.2][]
+* [llhttp 8.1.0][]
+* [minimatch 9.0.0][]
+* [nghttp2 1.53.0][]
+* [nghttp3 0.7.0][]
+* [ngtcp2 0.8.1][]
+* [npm 9.6.7][]
+* [openssl 3.0.8][]
+* [postject 1.0.0-alpha.6][]
+* [simdutf 3.2.9][]
+* [undici 5.22.1][]
+* [uvwasi 0.0.16][]
+* [V8 11.3.244.8][]
```

Comment:
Someone please send a PR to https://github.com/nodejs/node-core-utils/tree/main/lib/update-v8 to automatically change this markdown file during major V8 updates.

Reply:
Here is the PR https://github.com/nodejs/node-core-utils/pull/699